config/locales/es.yml

```diff
@@ -170,3 +170,4 @@ es:
   editor: "Editor"
   console: "Consola"
   type_expressions_here: "Acá podés escribir expresiones para probar tu solución"
+  repeat_pending: "Terminar pendientes"
```

Comment:
Ok, maybe you are right. 

We are using here [repeat](http://fortawesome.github.io/Font-Awesome/icon/repeat/) icon. What about simply using [next icon](http://fortawesome.github.io/Font-Awesome/icon/chevron-right/) like the rest?